src/cc2olx/olx.py

```diff
@@ -163,7 +163,10 @@ def _add_olx_nodes(self, element, course_data, tags):
 
             for child in children:
                 if "title" in element_data:
-                    child.setAttribute("display_name", element_data["title"])
+                    if "%" in element_data["title"]:
+                        logger.warning("Removing invalid percent character from title, %s.", element_data["title"])
+                    formatted_title = element_data["title"].replace("%", "")
```

Comment:
in the bug case would it be more readable to remove the % or replace it with a space or underscore or dash?

Reply:
So my thinking behind it was more to have it be as close to the original text without requiring someone to manually change it. So for example:

`30-Lecture Video: Why the crossing over rate is never greater than 50%` would become `30-Lecture Video: Why the crossing over rate is never greater than 50`. To me that still feels logical since 50 is a rate and thus a percent could be implied (though it's a little weirder for the ice cream example below). Alie had suggested perhaps using the word percent instead of %, but I thought that this might be confusing if % were being used for something else (e.g. modulo operations).

Was your idea for replacing with something else for it to be more apparent for the user the need to manually fix in studio? I could see that working well with an underscore or dash, but a space might not work in that case as it would just create trailing space and this might be hard to see. For example `100% of people love ice cream` would become `100  percent of people love ice cream`.

Reply:
^ P.S. github is too smart here and automatically correcting the trailing space, but the idea being that there would be two spaces between 100 and percent instead of one.

Reply:
This is fine, you thought about it and removing it seemed like the best thing to do 👍

Reply:
Randomly driving by.  The nature of the error isn't mentioned in this PR, so I don't know what the root cause of this issue is.  I wondered why we simply could not encode certain characters in the title?

Reply:
Hey, good question @e0d . I've included more detail about the root cause in the description. Hopefully this context helps.

Reply:
Because there are other characters that could be invalid based on the regex, I'm going to try to see if I can just remove any invalid characters rather than the percent only.